ksql-engine/src/test/java/io/confluent/ksql/structured/SchemaKStreamTest.java

```diff
@@ -260,4 +295,252 @@ public void testGroupByMultipleColumns() {
 
     Assert.assertEquals(groupedSchemaKStream.getKeyField().name(), "TEST1.COL1|+|TEST1.COL0");
   }
+
+  @SuppressWarnings("unchecked")
+  @Test
+  public void shouldPerformStreamToStreamLeftJoin() {
+    final SchemaKStream secondSchemaKStream
```

Comment:
I think this is the same in all `StreamToStreamXxxxJoin()` tests. Perhaps refactor to a field or a method